### PR TITLE
Feat: J2lint GitHub action

### PR DESCRIPTION
**Pull request by ghost**
## Related Issue(s)

Fixes #<ISSUE ID>

## Component(s) name

`arista.avd.<role-name>`

### User Checklist

- N/A

### Repository Checklist

- [ ] My code has been rebased from devel before I start
- [ ] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [ ] My change requires a change to the documentation and documentation have been updated accordingly.
- [ ] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/plugins/modules/inventory_to_container.py [hunk 1]
```diff
@@ -404,7 +404,7 @@ def main():
         inventory=dict(type='str', required=True),
         container_root=dict(type='str', required=True),
         configlet_dir=dict(type='str', required=False),
-        configlet_prefix=dict(type='str', required=False),
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/plugins/modules/inventory_to_container.py:407]: unrelated change?

File: .github/workflows/j2lint.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+
+name: J2LINT
+
+on:
+  push:
+    branches: [ github-actions ]
```
Comment (ClausHolbechArista) [inline .github/workflows/j2lint.yml:6]: this should be updated to `devel`, same below.

**Review by ClausHolbechArista — CHANGES_REQUESTED** [review]
I think you started from another branch instead of devel. There are a lot of integration and unit tests in this PR. Please split it out, so this PR is only for the J2lint GitHub action

File: ansible_collections/arista/avd/plugins/modules/inventory_to_container.py (hunk 1 above)
Comment (Shivani-chourasiya) [reply ansible_collections/arista/avd/plugins/modules/inventory_to_container.py:407]: yes I tried it in different branch, once confirmed I can add it to devel

Comment (Shivani-chourasiya) [reply ansible_collections/arista/avd/plugins/modules/inventory_to_container.py:407]: pls check only J2lint.yml

Comment (Shivani-chourasiya) [reply ansible_collections/arista/avd/plugins/modules/inventory_to_container.py:407]: shall I add J2Lint.yml to devel branch?

Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/plugins/modules/inventory_to_container.py:407]: This PR is supposed to be what will be merged into devel. So the PR should only contain the relevant changes.

Comment (Shivani-chourasiya) [reply ansible_collections/arista/avd/plugins/modules/inventory_to_container.py:407]: Okay ,updating it. Sorry for the inconvenience.
